### PR TITLE
Update to system json required fields for preview category

### PR DESCRIPTION
**Pull request by rnaidu02**
Processor model and processor core count is not required in system_desc.json for preview systems. This PR is related to https://github.com/mlcommons/policies/issues/97

### CODE REVIEW

**Comment by rnaidu02** [comment]
@johntran-nv Can training WG review the PR and approve.

File: submission_rules.adoc [hunk 1]
```diff
@@ -435,9 +435,9 @@ The file <system_desc_id>.json should contain the following metadata describing
 | system_name | Yes | tpu-v3 | 8ball | 8ball
 | number_of_nodes | Yes | 1 | 1 | 1
 | host_processors_per_node | Yes | 1 | 2 | 2
-| host_processor_model_name | Yes | Intel Skylake | Intel Xeon Platinum 8164 | Intel Xeon Platinum 8164 
-| host_processor_core_count | Yes, or vcpu |  | 26 | 26
-| host_processor_vcpu_count | Yes, or core | 96 | |
+| host_processor_model_name | Yes ^1^| Intel Skylake | Intel Xeon Platinum 8164 | Intel Xeon Platinum 8164
```
Comment (rnaidu02) [inline submission_rules.adoc:438]: host_processor_model_name is required. However there is a flexibility to not including the model name for preview systems. Generic processor name is sufficient (ex: Intel Skylake) for preview submission

Comment (rnaidu02) [inline submission_rules.adoc:438]: host_processor_model_name is required. However there is a flexibility to not including the model name for preview systems. Generic processor name is sufficient (ex: Intel Skylake) for preview submission

**Comment by johntran-nv** [comment]
The Training WG approves.

**Pull request merged**